src/ReqHandler/ReqHandler.cpp

```diff
@@ -1,120 +1,78 @@
-#include "../../include/ReqHandler.hpp"
+#include "../../include/WebServ.hpp"
```

Comment:
```suggestion
#include "../../include/ReqHandler.hpp"
```

각 cpp는 해당하는 hpp를 가지고 오는게 좋은거 같습니니다.